From: Steven Chamberlain <steven@pyro.eu.org>
Subject: Fix build on GNU/kFreeBSD

* Still need to link with -ldl, unlike regular BSD
* Remove extraneous -llilv, which swig2.0 on BSD-like systems would
  fopen() even though it is a directory

---
 wscript |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- lilv.orig/wscript
+++ lilv/wscript
@@ -226,7 +226,7 @@ def build(bld):
     if bld.env.MSVC_COMPILER:
         libflags = []
         defines  = ['snprintf=_snprintf']
-    elif sys.platform.find('bsd') > 0:
+    elif sys.platform.find('bsd') > 0 and sys.platform.find('gnu') < 0:
         lib = []
 
     # Shared Library
@@ -351,7 +351,7 @@ def build(bld):
                   source     = 'bindings/lilv.i',
                   target     = 'bindings/_lilv',
                   includes   = ['..'],
-                  swig_flags = '-c++ -python -Wall -I.. -llilv -features autodoc=1',
+                  swig_flags = '-c++ -python -Wall -I.. -features autodoc=1',
                   use        = 'liblilv')
         autowaf.use_lib(bld, obj, 'LILV')
 
